### PR TITLE
Feature/restructuring docker file

### PR DESCRIPTION
**Pull request by denieryd**
# Description

Restructured the dockerfile & fix a little error with sqlalchemy

# Checklist

- [x] I've generated a new bot from the async-box template and checked that everything works:
  - [x] Linters and tests have passed
  - [x] Bot answers on `/help` command
  - [x] Bot suggests available commands
- [x] I've written a changelog for PR change
- [x] I'll make a release when PR is approved

### CODE REVIEW

File: {{cookiecutter.bot_project_name}}/Dockerfile [hunk 1]
```diff
@@ -16,19 +16,24 @@ ARG GIT_HOST=""
...
+    echo "${NON_ROOT_USER} ALL = NOPASSWD: /usr/sbin/update-ca-certificates" > /etc/sudoers.d/express_bot && \
+    apt-get clean autoclean && \
+    apt-get autoremove --yes && \
+    rm -rf /var/lib/{apt,dpkg,cache,log}/
+
 # Poetry can't read password to download private repos
-RUN echo -e "machine ${GIT_HOST}\nlogin ${GIT_LOGIN}\npassword ${GIT_PASSWORD}" > ~/.netrc && \
-    apt-get update && \
-    apt-get install -y git sudo && \
+RUN apt-get install -y git && \
```
Comment (vonabarak) [inline {{cookiecutter.bot_project_name}}/Dockerfile]: Suggest to move `apt-get install -y git` into first RUN and remove clearning apt cache from second RUN.

File: {{cookiecutter.bot_project_name}}/Dockerfile [hunk 2]
```diff
@@ -16,19 +16,24 @@ ARG GIT_HOST=""
 ARG GIT_PASSWORD=${CI_JOB_TOKEN}
 ARG GIT_LOGIN="gitlab-ci-token"
 
+RUN apt-get update && \
+    apt-get install -y sudo && \
```
Comment (vonabarak) [inline {{cookiecutter.bot_project_name}}/Dockerfile]:
```suggestion
    apt-get install -y sudo git && \
```

File: {{cookiecutter.bot_project_name}}/Dockerfile (hunk 1 above)
Comment (denieryd) [reply {{cookiecutter.bot_project_name}}/Dockerfile]: git is not used in first run, does it make sense to move it there?

Rmove cleaning apt cache - you are about 
`apt-get clean autoclean && \
 apt-get autoremove --yes && \
 rm -rf /var/lib/{apt,dpkg,cache,log}/ && \
 ` ?

Comment (vonabarak) [reply {{cookiecutter.bot_project_name}}/Dockerfile]: > git is not used in first run, does it make sense to move it there?

I think it just looks more clear, when all packages are installed at once.

> you are about apt-get clean autoclean && \ apt-get autoremove --yes && \ rm -rf /var/lib/{apt,dpkg,cache,log}/ && \

Yes.

Comment (denieryd) [reply {{cookiecutter.bot_project_name}}/Dockerfile]: fixed

**Review by vonabarak — APPROVED** [review]

**Review by Opti213 — APPROVED** [review]

**Pull request merged**